AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs

```diff
@@ -153,27 +152,24 @@ public IActionResult Register()
         // GET: /Account/ConfirmEmail
         [HttpGet]
         [AllowAnonymous]
-        public async Task<IActionResult> ConfirmEmail(string userId, string code)
+        public async Task<IActionResult> ConfirmEmail(string userId, string token)
         {
-            if (userId == null || code == null)
-            {
+            if (userId == null || token == null)
                 return View("Error");
-            }
+

```

Comment:
@BillWagner crap. I've been coding all along to take the curly braces out of single line statement clauses. Going forward I'll adhere to that approach